spark-commons/src/test/scala/za/co/absa/spark/commons/OncePerSparkSessionTest.scala

```diff
@@ -48,4 +47,29 @@ class OncePerSparkSessionTest extends AnyFunSuite with MockitoSugar with SparkTe
     assert(libraryBInitCounter == 2)
   }
 
+  test("should return true if the library is registered successfully and false if not, and if spark session hasn't started"){
+    val anotherSpark: SparkSession =  mock[SparkSession]
```

Comment:
**Minor:** You don't need the mock here. You can just use the `spark` from the parent `SparkTestBase`